source/core/NumericString.cpp

```diff
@@ -1291,6 +1293,39 @@ IntMax ScanIntValues(char formatChar, char* beginPointer, char** endPointer)
     return intValue;
 }
 
+std::string validTrueValues[5] = { "y", "t", "true", "on", "yes" };
+std::string validFalseValues[4] = { "f", "false", "off", "no" };
+
+Boolean ScanBooleanValue(char* beginPointer, char** endPointer)
+{
+    std::string toLowered = std::string(beginPointer);
+    *endPointer = beginPointer;
+
+    std::transform(toLowered.begin(), toLowered.end(), toLowered.begin(), ::tolower);
```

Comment:
I actually don't think it's a good idea to be constructing std::strings in a run-time operation for performance reasons.  None of the other string parsing functions do this.

Reply:
+1. Why not use one of the SubString::Compare() variants that can be called with ignoring case.